src/Rules.Framework/Evaluation/ValueEvaluation/StartsWithOperatorEvalStrategy.cs

```diff
@@ -0,0 +1,20 @@
+namespace Rules.Framework.Evaluation.ValueEvaluation
+{
+    using System;
+
+    internal class StartsWithOperatorEvalStrategy : IOneToOneOperatorEvalStrategy
+    {
+        public bool Eval(object leftOperand, object rightOperand)
+        {
+            if (leftOperand is string)
```

Comment:
Can you also please validate `rightOperand`?

Reply:
validation added, please take a look